albumentations/core/utils.py

```diff
@@ -10,3 +11,74 @@ def format_args(args_dict):
             v = "'{}'".format(v)
         formatted_args.append('{}={}'.format(k, v))
     return ', '.join(formatted_args)
+
+
+class Params:
+    __metaclass__ = ABCMeta
```

Comment:
If I am not mistaken this syntax works only for Python 2. For Python 3 you have to pass a metaclass as a `metaclass` argument to the class definition. To make this class compatible both with Python 2 and Python 3 you can use `with_metaclass` (example - https://github.com/albu/albumentations/blob/master/albumentations/core/transforms_interface.py#L45)

Reply:
it works, but added with_metaclass to be 100% sure